### DISCUSSION TITLE
How do I save settings permanently?

### DISCUSSION THREAD

**Discussion by RedShark82** [lobehub/lobehub#1275] [General]
I have installed lobe-chat via Docker on my home server. However, it seems that you cannot specify a volume, which is why my settings are gone after an update. I have seen that you can download a configuration file, but it can't be meant that you have to import it again after every update. So how can I save my settings permanently?

**Comment by arvinxx** [comment]
The setting is persisted in your browser db. So It won't missing after you upgrade docker version

**Comment by RedShark82** [comment]
Thanks for the explanation, then it wasn't the update but the fact that I was using a different PC/browser. Saving the data in a volume would be better from my point of view.

**Reply by arvinxx** [reply]
It's not for all users. For example, If you just share your site with friends, they will immediately has  your setting If It's setted in docker volume.It's not a goodway

**Reply by weselben** [reply]
i dont agree actually, it would be way smarter to just add roles based access, and then store it in volume or in browser based on something like "admin settings"

**Comment by RedShark82** [comment]
But if that's what it is intended for, wouldn't a user administration be best?

Don't get me wrong. I think Lobe-Chat is really great also because it supports GPT4-Turbo which many open source UI's don't do so far. But for me, using it on 5-6 different clients, the way settings are saved is a bit frustrating.

**Reply by jsuelwald** [reply]
Yes, it would be. I want to try lobe-chat but if this isn't possible to store i think i'll stick with open-webui

**Reply by mamachanko** [reply]
Have a look at [deploying LobeChat w/ a server-side database](https://lobehub.com/docs/self-hosting/server-database). That's exactly designed for your use-case where you want to conversations to carry across devices.

**Comment by arvinxx** [comment]
you can try the lobe-chat-database now. It support database persistence
